indico/modules/events/sessions/templates/display/session_display.html

```diff
@@ -16,7 +16,13 @@
 {% endblock %}
 
 {% block info %}
-    <div class="time-data">
```

Comment:
```
>>> Event.query.filter(db.or_(Event.stylesheet.ilike(f'%{x}%') for x in ('time-data', 'location-data', 'board-data')), ~Event.is_deleted).count()
7
```

Check the custom CSS of these events in our production instance to see whether this change will break things. If yes we may want to consider leaving the old classes in addition to the new ones (or just keeping the old ones instead of replacing them with a generic one)...

Reply:
Checked those 7, they seem fine to me. Since those entries are from 2018/19, I don't think it's worth keeping.

Reply:
I guess it also depends on whether we want to give people a way to target those elements in custom CSS or not..

Reply:
But TBH I don't think it's needed. These classes aren't used on the main conference page, are they? Because *there* I think that'd be more important.. I'm not aware of people styling contribution/session pages...

Reply:
No, they aren't, only the individual pages.